### ISSUE TITLE
Analyzing anchors... 

### ISSUE THREAD

**Issue by Gaoang112** [ultralytics/yolov5#1087]
When I trained my own data set show "Analyzing anchors... 非法指令 (核心已转储)",how to solve? Thank you for your.

**Comment by Gaoang112** [comment]
Model Summary: 263 layers, 2.14899e+07 parameters, 2.14899e+07 gradients

Optimizer groups: 86 .bias, 94 conv.weight, 83 other
Scanning images: 100%|████████████████████| 1406/1406 [00:00<00:00, 9298.65it/s]
Scanning labels coco/labels.cache (1406 found, 0 missing, 0 empty, 0 duplicate, for 1406 images): 1406it [00:00, 14052.84it/s]
Scanning labels coco/labels.cache (1055 found, 0 missing, 0 empty, 0 duplicate, Scanning labels coco/labels.cache (1406 found, 0 missing, 0 empty, 0 duplicate, for 1406 images): 1406it [00:00, 11093.06it/s]

Analyzing anchors... 非法指令 (核心已转储)

**Comment by NanoCode012** [comment]
@Gaoang112 , nothing is wrong. Just let it run. It's trying to find the best anchors for you.

**Comment by rafael-junio** [comment]
ython3 train.py --img 640 --batch 8 --epochs 300 --data ./data/semalo.yaml --cfg ./models/yolov5x.yaml
Using CUDA device0 _CudaDeviceProperties(name='GeForce GTX 1070', total_memory=8118MB)

Namespace(adam=False, batch_size=8, bucket='', cache_images=False, cfg='./models/yolov5x.yaml', data='./data/semalo.yaml', device='', epochs=300, evolve=False, global_rank=-1, hyp='data/hyp.scratch.yaml', image_weights=False, img_size=[640, 640], local_rank=-1, logdir='runs/', multi_scale=False, name='', noautoanchor=False, nosave=False, notest=False, rect=False, resume=False, single_cls=False, sync_bn=False, total_batch_size=8, weights='yolov5s.pt', workers=8, world_size=1)
Start Tensorboard with "tensorboard --logdir runs/", view at http://localhost:6006/
Hyperparameters {'lr0': 0.01, 'lrf': 0.2, 'momentum': 0.937, 'weight_decay': 0.0005, 'warmup_epochs': 3.0, 'warmup_momentum': 0.8, 'warmup_bias_lr': 0.1, 'giou': 0.05, 'cls': 0.5, 'cls_pw': 1.0, 'obj': 1.0, 'obj_pw': 1.0, 'iou_t': 0.2, 'anchor_t': 4.0, 'fl_gamma': 0.0, 'hsv_h': 0.005, 'hsv_s': 0.05, 'hsv_v': 0.05, 'degrees': 0.0, 'translate': 0.1, 'scale': 0.5, 'shear': 0.1, 'perspective': 0.0, 'flipud': 0.0, 'fliplr': 0.4, 'mosaic': 0.1, 'mixup': 0.0}

             
[...]
Model Summary: 407 layers, 8.88576e+07 parameters, 8.88576e+07 gradients

Transferred 59/802 items from yolov5s.pt
Optimizer groups: 134 .bias, 142 conv.weight, 131 other
Scanning labels ../semalo-dataset/labels/train.cache (8724 found, 0 missing, 9 empty, 0 duplicate, for 8733 images): 8733it [00:01, 6608.00it/s]
Scanning labels ../semalo-dataset/labels/val.cache (2493 found, 0 missing, 1 empty, 0 duplicate, for 2494 images): 2494it [00:00, 5728.03it/s]

Analyzing anchors... Illegal instruction

Same problem here

**Comment by NanoCode012** [comment]
Hi @rafael-junio , I would ask that you try to do a quick training on coco128 to make sure nothing else is wrong.

Another thing I noticed was that you would like to use the `5x` model for your training, correct? However, from your `opt` namespace, you are using the `5s` weights. You should just pass `weights yolov5x.pt` too. I think that’s why not many layers were transferred.

See for more info https://github.com/ultralytics/yolov5/pull/680#issue-465107877

**Comment by glenn-jocher** [comment]
@rafael-junio yes, pretrained weights will be used that match layer name and shape, which is not many layers between your --weights yolov5s.pt and your --cfg yolov5x.pt. You can train from scratch with --weights '' or you can train from --weights yolov5x.pt

**Comment by rafael-junio** [comment]
> Hi @rafael-junio , I would ask that you try to do a quick training on coco128 to make sure nothing else is wrong.
> 
> Another thing I noticed was that you would like to use the `5x` model for your training, correct? However, from your `opt` namespace, you are using the `5s` weights. You should just pass `weights yolov5x.pt` too. I think that’s why not many layers were transferred.
> 
> See for more info [#680 (comment)](https://github.com/ultralytics/yolov5/pull/680#issue-465107877)

Tried all weights combinations. The problem probably lies in my CPU, as it doesn't supports AVX instructions.
